gherkin/java/src/main/java/gherkin/GherkinLanguageConstants.java

```diff
@@ -1,10 +1,15 @@
 package gherkin;
 
+import java.util.regex.Pattern;
+
 public interface GherkinLanguageConstants {
     String TAG_PREFIX = "@";
     String COMMENT_PREFIX = "#";
     String TITLE_KEYWORD_SEPARATOR = ":";
+    Pattern TITLE_KEYWORD_SEPARATOR2 = Pattern.compile(":|：");
```

Comment:
Please remove the old `TITLE_KEYWORD_SEPARATOR` and rename `TITLE_KEYWORD_SEPARATOR2` to `TITLE_KEYWORD_SEPARATOR`